Formula/lightgbm.rb

```diff
@@ -0,0 +1,34 @@
+class Lightgbm < Formula
+  desc "Fast, distributed, high performance gradient boosting framework"
+  homepage "https://github.com/microsoft/LightGBM"
+  url "https://github.com/microsoft/LightGBM/archive/v2.2.3.tar.gz"
+  sha256 "4a6414e808b343a784f0ee805ac723c094488f9e9a951dd3f709dc31ffb4ea4c"
+
+  depends_on "cmake" => :build
+  depends_on "libomp"
+
+  def install
+    mkdir "build" do
+      libomp = Formula["libomp"]
+      args = std_cmake_args
+      args << "-DOpenMP_C_FLAGS=\"-Xpreprocessor -fopenmp -I#{libomp.opt_include}\""
+      args << "-DOpenMP_C_LIB_NAMES=omp"
+      args << "-DOpenMP_CXX_FLAGS=\"-Xpreprocessor -fopenmp -I#{libomp.opt_include}\""
+      args << "-DOpenMP_CXX_LIB_NAMES=omp"
+      args << "-DOpenMP_omp_LIBRARY=#{libomp.opt_lib}/libomp.dylib"
+
+      system "cmake", *args, ".."
+      system "make"
+    end
+    lib.install "lib_lightgbm.so"
+    bin.install "lightgbm"
```

Comment:
Is there no `make install` target?

Reply:
Hey @fxcoudert, thanks for the feedback. There was a `make install` target, so switched to that.